core/src/test/resources/h2/schema-h2.sql

```diff
@@ -246,3 +246,20 @@ CREATE TABLE IF NOT EXISTS `tag_relation_meta` (
     KEY `idx_tid` (`tag_id`),
     KEY `idx_mid` (`metadata_object_id`)
     ) ENGINE=InnoDB;
+
+CREATE TABLE IF NOT EXISTS `owner_meta` (
+    `id` BIGINT(20) UNSIGNED NOT NULL AUTO_INCREMENT COMMENT 'auto increment id',
+    `metalake_id` BIGINT(20) UNSIGNED NOT NULL COMMENT 'metalake id',
+    `owner_id` BIGINT(20) UNSIGNED NOT NULL COMMENT 'owner id',
+    `owner_type` VARCHAR(64) NOT NULL COMMENT 'owner type',
+    `entity_id` BIGINT(20) UNSIGNED NOT NULL COMMENT 'entity id',
+    `entity_type` VARCHAR(64) NOT NULL COMMENT 'entity type',
```

Comment:
owner is binding to a securable object or metadata object, you'd better use such name, not "entity"

Reply:
I have changed this to metadata object.